Formula/c-toxcore.rb

```diff
@@ -0,0 +1,38 @@
+class CToxcore < Formula
+  desc "C library implementing the Tox peer to peer network protocol"
+  homepage "https://tox.chat/"
+  url "https://github.com/TokTok/c-toxcore/releases/download/v0.2.18/c-toxcore-0.2.18.tar.gz"
+  sha256 "f2940537998863593e28bc6a6b5f56f09675f6cd8a28326b7bc31b4836c08942"
+  license "GPL-3.0-or-later"
+  head "https://github.com/TokTok/c-toxcore.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libconfig"
+  depends_on "libsodium"
+  depends_on "libvpx"
+  depends_on "opus"
+
+  def install
+    system "cmake", "-S", ".", "-B", "_build", *std_cmake_args
+    system "make", "-C", "_build"
+    system "make", "-C", "_build", "install"
```

Comment:
Why not
```suggestion
    system "cmake", "--build", "_build"
    system "cmake", "--install", "_build"
```
?

Reply:
I'm familiar with `make` but not `cmake`, and this way using `make` was what was in toxcore's documentation, so that's the reason why I had it this way, but you're right that this is simpler and it works great. Applying suggestion!